ale_linters/proto/protoc-gen-lint.vim

```diff
@@ -0,0 +1,10 @@
+" Author: Jeff Willette <jrwillette88@gmail.com>
+" Description: run the protoc-gen-lint plugin for the protoc binary
+
+call ale#linter#Define('proto', {
+\   'name': 'protoc-gen-lint',
+\   'output_stream': 'stderr',
+\   'executable': 'protoc',
+\   'command': 'protoc --lint_out=. *.proto',
```

Comment:
It looks like `*.proto` is for checking files in the current directory. That will be expanded in the shell. Does this program read the file from stdin at all?